lambdas/postcode-lookup/src/main/java/uk/gov/di/ipv/cri/address/api/service/PostcodeLookupService.java

```diff
@@ -73,7 +79,18 @@ public List<CanonicalAddress> lookupPostcode(String postcode)
         this.validatePostCode(postcode);
         // Create our http request
         HttpRequest request = createHttpRequest(postcode);
+
+        long startTime = System.nanoTime();
```

Comment:
I think this is nicer

```
Instant start = Instant.now();
doSomething()
Instant start = Instant.now();
Duration.between(start, end).toMillis()
```

Reply:
Maybe but there's a fundamental problem with `Instant` which is that it's based on the system clock so it can be affected by factors such as manual adjustments or daylight savings changes. I chose `System.nanoTime()` because it's based on a monotonous clock that is not affected by system time changes, making it more reliable for measuring elapsed time. It is designed to measure elapsed time and is guaranteed to be monotonic, meaning it always moves forward and is not subject to changes in the system clock.

The same issues occurs with `System.currentTimeMillis()` and it was one of the factors that was considered in check-hmrc hence why we used `performance.now()` there.